src/main/java/com/google/gerrit/extensions/api/changes/ChangeApi.java

```diff
@@ -273,6 +275,11 @@ public void topic(String topic) throws RestApiException {
     }
 
     @Override
+    public List<AccountInfo> listReviewers() throws RestApiException {
```

Comment:
GitHub seems to have interpreted my comment here. Please return List[ReviewerInfo] here (replace [ with < here).